tests/test_cli.py

```diff
@@ -0,0 +1,167 @@
+from pathlib import Path
+
+import click.testing
+import pytest
+
+from csv2docx import cli
+
+
+default_outpath = Path.cwd() / "output"
+
+
+@pytest.fixture
+def runner():
+    return click.testing.CliRunner()
+
+
+# not using a tmp_path to test default output folder
+def test_basic_succeeds(runner):
+    result = runner.invoke(
+        cli.main,
+        [
+            "--template",
+            "tests/data/example.docx",
+            "--data",
+            "tests/data/example.csv",
+            "--name",
+            "NAME",
+        ],
```

Comment:
__Note:__ I have not tested the above so am uncertain if it'll work, but it gives an idea of what regarding using a fixture to reduce code duplication 👍

Reply:
Perhaps an alternative (imho cleaner) approach to the above would be using [`parametrize`](https://docs.pytest.org/en/latest/parametrize.html). Open to debate 👍

Reply:
re:parametrizing -- I looked into this and it is not as simple to implement this for the CLI as [`pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html) does not accept fixtures. See [this PyCon2020 video for an overview parameterized testing](https://us.pycon.org/2020/schedule/presentation/172/). As an example, here's what it might look like for testing the library under successful conditions:

```python

def library_success_loads():
    return  test_cases = [                                                                 
      ("tests/data/example.csv", "tests/data/example.docx", "NAME", "output", ";"),
      ("tests/data/example.csv", "tests/data/example.docx", "NAME", "nested/folder", ";"),
]                                                                              

@pytest.mark.parametrize("data, template, name, path, delimiter", test_cases)  
def test_library_convert_success(data, template, name, path, delimiter):               
     result = c2d.convert(data, template, name, path, delimiter)      
     # Note: not sure what to assert as convert success case returns None.     
     # which makes this comparison feel on pythonic  
     assert result  == None
```

Reply:
Implemented the fixture, but moved the discussion of parametrizing to #23